api/basic/schema.ts

```diff
@@ -0,0 +1,50 @@
+export interface Response {
+  statusCode: number
+  body: any
+}
+
+export interface Pin {
+  cid: string
+  status: 'queued' | 'pinning' | 'pinned' | 'failed' | 'unpinned'
+  created: string
+}
+
+export interface ClusterStatusResponse {
+  'cid': string
+  'name': ''
+  'allocations': []
+  'origins': []
+  'created': string
+  'metadata': null
+  'peer_map': {
+    '12D3KooWArSKMUUeLk3z2m5LKyb9wGyFL1BtWCT7Gq7Apoo77PUR': {
```

Comment:
What if we change peerId? I would just set a Record here with string as key

Reply:
yah, but it is a fake cluster peerID, there is no reason to change it. I agree it's suprising to see it hardcoded here, but it's only their to make the response the right shape. We might change the ipfs peer id / addr, but there is no reason to change the fake cluster peer id.

Reply:
this is an Interface, we should be able to just create a new interface for PeerMapValue and do

```ts
`peer_map`: Record<string, PeerMapValue>
```

What do you think?

Reply:
Fixed as you suggested!